test/integration/projects_int_test.go

```diff
@@ -75,6 +77,87 @@ func (suite *ProjectsIntegrationTestSuite) TestJSON() {
 	//AssertValidJSON(suite.T(), cp) // list is too large to fit in terminal snapshot
 }
 
+func (suite *ProjectsIntegrationTestSuite) TestEdit_Name() {
```

Comment:
These two tests will fail sporadically due to race conditions when any two platforms are running them at the same time. Would it be reasonable to have platform-specific project names for these tests? (e.g. "Edit-Test" + runtime.GOOS).

Reply:
Very good point! Thanks for catching that.